src/main/java/com/dev/cinema/model/MovieSession.java

```diff
@@ -16,10 +16,12 @@
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
-    @OneToOne
+    @ManyToOne
     private Movie movie;
     @ManyToOne
+    @JoinColumn(name = "cinema_hall")
```

Comment:
let's name it `cinema_hall_id` if there is `movie_id` column created for Movie automatically by Hibernate